website/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ESNext",
```

Comment:
This allow to use new feature, Next doesn't use tsconfig to build the project, so there is no runtime change